cves/CVE-2013-1896.yml

```diff
@@ -99,11 +115,14 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
-  contest:
+  answer: |
+    The vulnerability was discovered and a fix was proposed (and accepted) by
```

Comment:
Maybe some clarification could bse useful here. Right now, it sounds like the same guy who discovered the fix also wrote the fix and then accepted his own fix(?) Probably not the case, but that's how the sentence reads.

Reply:
Reworded to make this less confusing